src/plugin/ipc/file/fileconnection.cpp

```diff
@@ -371,7 +368,15 @@ FileConnection::refill(bool isRestart)
         }
       }
       tempfd = openFile();
+    } else if (_fcntlFlags & O_WRONLY) {
+      // File doesn't exist. If it's a WRONLY file, we'll create a new one.
+      tempfd = _real_open(_path.c_str(), O_CREAT|O_WRONLY|O_TRUNC);
+      ASSERT_NE(-1, tempfd);
```

Comment:
Isn't `JASSERT(tmpfd != -1)` easier to read than `ASSERT_NE()`?  Also, it seems confusing that we have two variants:  `JASSERT` and `ASSERT_XX`, in addition to the normal `assert()`.